src/root/views/account.js

```diff
@@ -791,6 +792,12 @@ class Account extends React.Component {
 
   render () {
     const { strings } = this.context;
+    const pending = this.isPending(this.props.profile);
+    if (pending) {
+      return (
+        <NewGlobalLoading />
+      );
+    }
```

Comment:
we can figure out pending inline as
```
const pending = this.props.profile?.fetching;
```

And put the global loading without blocking the view i.e. inside `App`
```
<App className='page--account'>
    { pending && <NewGlobalLoading /> }
    ...
```

Reply:
Same as above.